core/secure_dt.mk

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (c) 2015-2018, ARM Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Any particular reason why this is BSD-3-Clause instead of the BSD-2-Cause that we normally use?

Reply:
And since you're at it, "ARM" should be "Arm" since they rebranded a year ago or so.

Reply:
The original code comes from TF-A under this very licensing: [build_macros.mk](https://github.com/ARM-software/arm-trusted-firmware/blob/master/make_helpers/build_macros.mk#L2)